index.js

```diff
@@ -25,14 +27,31 @@ functions.handler = async (event, context, callback) => {
     // Get the record
     const record = event.Records[0];
 
-    // Log fit for Metric Filtering in CloudWatch
-    console.log('[S3:ObjectCreated]', record.s3.bucket.name.replace(/-/g, '_'), record.s3.object.key);
-
     // Get the bucket and key
     const bucket = record.s3.bucket.name;
-    const key = decodeURIComponent(record.s3.object.key.replace(/\+/g, ' '))
+    const key = decodeURIComponent(record.s3.object.key.replace(/\+/g, ' '));
     console.log(`Received event for [${bucket}/${key}]`);
 
+    // Send SNS message for a National News Tonight upload.
+    // In future, if this is requested for other programs,
+    // use switch cases to setup SNS parameters
+    if(key.includes("nationalnewstonight")) {
```

Comment:
I think it's safer to convert this if block into one statement with a try-catch around it here

Reply:
```
try {
  postToSNS()
} catch {
 console.log
} 
```

Reply:
postToSNS() does use a try-catch block, so any of its errors would be caught.